src/analysis/retail/mage/arcane/guide/ArcaneBarrage.tsx

```diff
@@ -101,33 +113,32 @@ class ArcaneBarrageGuide extends Analyzer {
       }
 
       const lowMana = ab.mana && ab.mana < LOW_MANA_THRESHOLD;
-      if (this.isSpellslinger && (ab.intuition || aethervisionStacks) && lowMana) {
+      if (this.isSpellslinger && (ab.intuition) && lowMana) {
         tooltipItems.push({ perf: QualitativePerformance.Good, detail: `Below 70% Mana` });
       }
 
       const lowHealth = ab.health && ab.health < EXECUTE_HEALTH_PERCENT;
-      if (this.isSpellslinger && (ab.intuition || aethervisionStacks) && lowHealth) {
+      if (this.isSpellslinger && (ab.intuition) && lowHealth) {
```

Comment:
your PR description indicates that you're working on sunfury but these changes are spellslinger-only. is that intended?